Assignments/SCF_Session-2/SCF_Assignment-1/src/HexCalculator.java

```diff
@@ -0,0 +1,122 @@
+
+public class HexCalculator implements HexCalcMethods{
+
+	@Override
+	public String add(String firstNumber, String secondNumber) // adds two hex numbers and returns hexnumber 
+	{
+		int sum=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		sum=number1+number2;
+		
+		return (decimalToHex(String.valueOf(sum)));
+	}
+
+	@Override
+	public String subtract(String firstNumber, String secondNumber) // subtracts two hex numbers and returns hexnumber 
+	{
+		int difference=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		difference=number1+number2;
+		
+		return (decimalToHex(String.valueOf(difference)));
+	}
+
+	@Override
+	public String multiply(String firstNumber, String secondNumber) // multiply two hex numbers and returns hexnumber 
+	{
+		int product=0;
+		int number1=Integer.parseInt(hexToDecimal(firstNumber));
+		int number2=Integer.parseInt(hexToDecimal(secondNumber));
+		product=number1+number2;
```

Comment:
@meta-vaibhav-gautam Don't copy and paste. You should make sure what calculation you are doing.